airflow/models/abstractoperator.py

```diff
@@ -17,14 +17,18 @@
 # under the License.
 
 import datetime
-from typing import TYPE_CHECKING, Any, Callable, Collection, Dict, List, Optional, Set, Type, Union
+from typing import TYPE_CHECKING, Any, Callable, Collection, Dict, Iterable, List, Optional, Set, Type, Union
+
+from sqlalchemy.orm import Session
```

Comment:
Type checking only import?

Reply:
Yes, but if `Session` is ever passed in as an argument this class is already in memory anyway, making this import basically free.